### DISCUSSION TITLE
Creating a list with a question mark or emoji might be causing a 500 error

### DISCUSSION THREAD

**Discussion by benjavicente** [community/community#8616] [Other Feature Feedback, Questions, & Ideas]
I created multiple lists, and when I tried to create one named `🖼 ????` (as a placeholder) I got an error message mentioning that the list couldn't be saved. When I got back to the stars tab, the 505 error page has displayed, and since then, I cannot see my stars.

**Comment by cheshire137** [comment]
Hi, thanks for reporting this! I think this is the same problem that another user experienced in https://github.com/github/feedback/discussions/8617. We working on a fix and hope to have it out today.
